packages/react/src/BreakoutBanner/BreakoutBanner.tsx

```diff
@@ -0,0 +1,227 @@
+          : {
+              [`--brand-BreakoutBanner-narrow-${property}`]: processBackgroundValue(value?.narrow, property),
+              [`--brand-BreakoutBanner-regular-${property}`]: processBackgroundValue(value?.regular, property),
+              [`--brand-BreakoutBanner-wide-${property}`]: processBackgroundValue(value?.wide, property),
+            },
+      [processBackgroundValue],
+    )
+
+    const backgroundStyles = useMemo(
+      () => ({
+        ...createStyles('background-color', backgroundColor),
+        ...createStyles('background-image-src', backgroundImageSrc),
+        ...createStyles('background-image-position', backgroundImagePosition),
+        ...createStyles('background-image-size', backgroundImageSize),
+      }),
+      [backgroundColor, backgroundImageSrc, backgroundImagePosition, backgroundImageSize, createStyles],
+    )
+
+    const hasHeading = useMemo(
+      () => React.Children.toArray(children).some(child => React.isValidElement(child) && child.type === _Heading),
+      [children],
+    )
+
+    if (!hasHeading && (process.env.NODE_ENV === 'development' || process.env.NODE_ENV === 'test')) {
+      // eslint-disable-next-line no-console
+      console.warn('The `BreakoutBanner` component expects a `BreakoutBanner.Heading` element to be specified.')
+    }
+
+    return (
+      <div
+        ref={ref}
+        className={clsx(styles.BreakoutBanner, className)}
+        style={{...backgroundStyles, ...style}}
+        {...props}
+      >
+        <div className={clsx(styles['BreakoutBanner-container'])}>
+          <div
+            className={clsx(
+              styles['BreakoutBanner-content'],
+              align === 'center' && styles['BreakoutBanner-content--center'],
+            )}
+          >
+            {LeadingVisual && <div className={clsx(styles['BreakoutBanner-leadingVisual'])}>{LeadingVisual}</div>}
+            {children}
+          </div>
+        </div>
+      </div>
+    )
+  },
+)
+
+const defaultHeadingLevel = 'h3'
+
+type BreakoutBannerHeadingProps = BaseProps<HTMLHeadingElement> & {
+  children: React.ReactNode | React.ReactNode[]
+} & HeadingProps
+
+const defaultHeadingSize = '4'
+
+const _Heading = forwardRef(
+  (
+    {as = defaultHeadingLevel, size = defaultHeadingSize, className, children, ...props}: BreakoutBannerHeadingProps,
+    ref: Ref<HTMLHeadingElement>,
+  ) => {
+    return (
+      <Heading ref={ref} className={clsx(styles['BreakoutBanner__heading'], className)} size={size} as={as} {...props}>
+        {children}
+      </Heading>
+    )
+  },
+)
+
+type BreakoutBannerDescriptionProps = BaseProps<HTMLParagraphElement> & {
+  children: React.ReactNode | React.ReactNode[]
+}
+
+const Description = forwardRef(
+  ({className, children, ...props}: BreakoutBannerDescriptionProps, ref: Ref<HTMLParagraphElement>) => {
+    return (
+      <Text ref={ref} className={clsx(styles['BreakoutBanner-description'], className)} as="p" {...props}>
```

Comment:
```suggestion
      <Text ref={ref} className={clsx(styles['BreakoutBanner-description'], className)} as="p" variant="muted" {...props}>
```

Could we use the `muted` variant here instead of setting the color in CSS?